pages/research_software_stories/acts_research_software_story.md

```diff
@@ -0,0 +1,177 @@
+## Community
+
+The ACTS community provides many resources to help onboard newcomers:
+
+- Contribution guidelines: A clear step-by-step guide for submitting code.
+
+- Interactive tutorials: Hands-on guides for configuring and running
+  tracking workflows.
+
+- Example framework: A sample project demonstrating how to use ACTS
+  algorithms in practice.
+
+In addition, regular workshops and training sessions help new members start
+using ACTS in their experiments.  New contributors are often mentored by
+experienced developers, ensuring knowledge transfer and continuous growth of
+the community.
+
+
+## Tools
+
+ACTS uses common tools to maintain quality and facilitate contributions:
+
+- Version control: Git, with repositories on GitHub for collaborative
+  development.
+
+- Continuous integration: Automated testing pipelines using GitHub Actions
+
+- Documentation: Documentation written with Sphinx and published on
+  ReadTheDocs.
+
+- Build system: CMake for consistent compilation across platforms.
+
+- Issue tracking: GitHub issue trackers for transparent bug reports and
+  feature requests.
+
+- Forums: Github Discussions, Mattermost messaging and e-mail lists for open
+  communication with the core development team.
+
+These tools streamline development, ensure reliability, and make it easier
+for new users to adopt and contribute to ACTS.
+
+
+## FAIR & Open
+
+ACTS adheres to FAIR principles.  The code is publicly available under the
+Mozilla Public License 2.0 (MPL-2.0), allowing broad use and collaboration. 
+Each release is archived with a DOI, so the software is easy to find and
+cite.  ACTS supports standard data formats and geometry descriptions, which
+ensures it is interoperable with various experimental frameworks.  By
+developing the toolkit in the open, ACTS fosters reuse across multiple
+research communities.
+
+
+## Documentation
+
+ACTS offers thorough documentation for both users and developers:
+
+- User guides: Detailed instructions for installation, setup, and usage.
+
+- Developer docs: Explanations of internal design, components, and APIs for
+  contributors.
+
+- Tutorials and examples: Step-by-step examples demonstrating common use
+  cases.
+
+The documentation is updated alongside the code to remain accurate and
+up-to-date.  Documentation contributions are valued as highly as code
+contributions, reflecting the importance of good documentation for ACTS's
+usability.
+
+
+## Sustainability
+
+ACTS is sustained by a strong community and multiple funding sources.  It
+benefits from institutional support (for example, CERN's EP R&D programme,
+the NSF-funded IRIS-HEP project in the US, the EU-funded AIDAInnova project,
+and CERN's Next Generation Triggers initiative).  Having contributors from
+many different experiments means ACTS isn't dependent on any single group or
+grant.  The project has clear governance and training practices to retain
+knowledge, which keeps ACTS resilient and ready to meet future needs.
+
```

Comment:
If the funding initiatives and CERNS EP R&D programme could have links - that would be great.